### ISSUE TITLE
Collaborative tags,  "Delete" Hint

### ISSUE THREAD

**Issue by SteveQi** [nextcloud/server#3967]
### Steps to reproduce
1.Files view list ,enter "Details" page
2.Delete an existing tag
3.This “Delete” hint  always exists

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system**: Linux 5e462a29cbca 3.10.0-514.10.2.el7.x86_64 #1 SMP Fri Mar 3 00:04:05 UTC 2017 x86_64

**Web server:** Apache (apache2handler)

**Database:** mysql 5.5.52

**PHP version:** 7.1.2
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, apache2handler, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, json, exif, mysqlnd, PDO, apcu, posix, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_sqlite, wddx, xmlreader, apc, zip, Phar, Zend OPcache

**Nextcloud version:** 11.0.2 (stable) - 11.0.2.7

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:** 

**Signing status:**
<details>
	<summary>Signing status</summary>

```
[]
```

</details>

**List of activated apps:**
<details>
	<summary>App list</summary>
```
Enabled:
 - dav: 1.1.1
 - federatedfilesharing: 1.1.1
 - files: 1.6.1
 - issuetemplate: 0.2.1
 - lookup_server_connector: 1.0.0
 - provisioning_api: 1.1.0
 - systemtags: 1.1.3
 - twofactor_backupcodes: 1.0.0
 - workflowengine: 1.1.1

Disabled:
 - activity
 - activitylog
 - admin_audit
 - admin_notifications
 - announcementcenter
 - apporder
 - audioplayer
 - bookmarks
 - calendar
 - checksum
 - comments
 - contacts
 - deck
 - direct_menu
 - encryption
 - external
 - federation
 - files_accesscontrol
 - files_automatedtagging
 - files_downloadactivity
 - files_external_sia
 - files_markdown
 - files_opds
 - files_pdfviewer
 - files_reader
 - files_retention
 - files_sharing
 - files_texteditor
 - files_trashbin
 - files_versions
 - files_videoplayer
 - firstrunwizard
 - gallery
 - githubmergetracker
 - gpxedit
 - gpxpod
 - keeweb
 - logreader
 - news
 - nextant
 - nextcloud_announcements
 - notes
 - notifications
 - ocr
 - ocsms
 - ojsxc
 - onlyoffice
 - ownpad
 - password_policy
 - piwik
 - previewgenerator
 - qownnotesapi
 - rainloop
 - registration
 - richdocuments
 - serverinfo
 - sharebymail
 - spreed
 - spreedme
 - survey_client
 - tasks
 - templateeditor
 - theming
 - twofactor_totp
 - twofactor_u2f
 - updatenotification
 - user_external
 - user_ldap
 - user_saml
 - weather
```
</details>

**The content of config/config.php:**
<details>
	<summary>Config report</summary>

```
{
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "localhost",
        "demo.nextcloud.com"
    ],
    "datadirectory": "\/var\/www\/html\/data",
    "overwrite.cli.url": "https:\/\/demo.nextcloud.com\/aabbcc",
    "dbtype": "mysql",
    "version": "11.0.2.7",
    "dbname": "nextcloud",
    "dbhost": "localhost",
    "dbport": "",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "Europe\/Berlin",
    "installed": true,
    "instanceid": "ochdhzpnxc96",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "overwritehost": "demo.nextcloud.com",
    "overwriteprotocol": "https",
    "overwritewebroot": "\/aabbcc",
    "log_rotate_size": "10485760",
    "mail_domain": "demo.nextcloud.com",
    "mail_smtpmode": "smtp",
    "mail_smtphost": "172.17.0.1",
    "trusted_proxies": {
        "1": "172.17.0.1"
    },
    "overwritecondaddr": "^172\\.17\\.0\\.1$",
    "htaccess.RewriteBase": "\/",
    "maintenance": false
}
```
</details>

**Are you using external storage, if yes which one:** files_external is disabled

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36

**Operating system:**

#### Web server error log
<details>
	<summary>Web server error log</summary>
	```
	Insert your webserver log here
	```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
	<summary>Nextcloud log</summary>
	```
	Insert your Nextcloud log here
	```
</details>

#### Browser log
<details>
	<summary>Browser log</summary>
	```
	Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
	```
</details>

**Comment by MorrisJobke** [comment]
cc @nickvergessen

**Comment by MorrisJobke** [comment]
Fix is in #3982
